pkg/graveler/graveler.go

```diff
@@ -1745,6 +1781,40 @@ func (g *KVGraveler) Commit(ctx context.Context, repositoryID RepositoryID, bran
 	return newCommitID, nil
 }
 
+func (g *KVGraveler) retryBranchUpdate(ctx context.Context, repositoryID RepositoryID, branchID BranchID, f BranchUpdateFunc) error {
+	const (
+		maxInterval = 5
+		maxElapsed  = 30
```

Comment:
units

Reply:
deleted, I don't understand what to unite.